chainlink-testing-framework/run-tests-binary/action.yml

```diff
@@ -0,0 +1,139 @@
+name: chainlink-testing-framework-binary-test-runner
+description: Used to run Chainlink tests only after the binary has been built with the build-tests action
+inputs:
+  artifacts_location:
+    required: false
+    description: Location of where error logs are written
+    default: ./logs
+  artifacts_name:
+    required: false
+    description: Name of the artifact to upload
+    default: test-logs
+  test_command_to_run:
+    required: true
+    description: The command to run the tests by calling your binary (note that -test.json is not supported)
+    # https://github.com/golang/go/issues/22996
+  cl_repo:
+    required: false
+    description: The Chainlink ecr repository to use
+    default: public.ecr.aws/z0b1w9r9/chainlink
+  cl_image_tag:
+    required: false
+    description: The chainlink image to use
+    default: develop
+  build_gauntlet_command:
+    required: false
+    description: How to build gauntlet if necessary
+    default: "false"
+  download_contract_artifacts_path:
+    required: false
+    description: Path where the contract artifacts need to be placed
+    default: "none"
+  token:
+    required: false
+    description: The GITHUB_TOKEN for the workflow
+    default: ${{ github.token }}
+  triggered_by:
+    required: true
+    description: The triggered-by label for the k8s namespace, required for cleanup
+    default: ci
+  cache_restore_only:
+    required: false
+    description: Only restore the cache, set to true if you want to restore and save on cache hit miss
+    default: "false"
+  cache_key_id:
+    required: false
+    description: Cache go vendors unique id
+    default: go
+  aws_registries:
+    required: false
+    description: AWS registries to log into for the test if needed
+  aws_role_duration_seconds:
+    required: false
+    default: "3600"
+    description: The duration to be logged into the aws role for
+  dockerhub_username:
+    description: Username for Docker Hub to avoid rate limits when pulling public images
+    required: false
+  dockerhub_password:
+    description: Password for Docker Hub to avoid rate limits when pulling public images
+    required: false
+  QA_AWS_REGION:
+    required: true
+    description: The AWS region to use
+  QA_AWS_ROLE_TO_ASSUME:
+    required: true
+    description: The AWS role to assume
+  QA_KUBECONFIG:
+    required: false
+    description: The kubernetes configuration to use
+  should_cleanup:
+    required: false
+    description: Whether to run the cleanup at the end, soak tests and such would not want to automatically cleanup
+    default: "false"
+  binary_name:
+    required: true
+    description: Name of the binary artifact to run
+    default: tests
+
+runs:
+  using: composite
+  steps:
+    - name: Setup Environment
+      uses: smartcontractkit/chainlink-github-actions/chainlink-testing-framework/setup-run-tests-environment@b664381f1437cb2b3cf63ddbc3fa663823d82586
```

Comment:
You probably want to set this to what the next tag version you will make when you merge otherwise this commit will go away eventually through github garbage collection caused by squash merge.

Reply:
Ya, I was wondering how to do that?